srcpkgs/whipper/template

```diff
@@ -1,22 +1,17 @@
 # Template file for 'whipper'
 pkgname=whipper
-version=0.9.0
-revision=3
+version=0.10.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools_scm"
 makedepends="libsndfile-devel python3-devel"
 depends="libcdio-paranoia cdrdao python3-gobject python3-musicbrainzngs
  python3-mutagen python3-requests python3-pycdio python3-discid
  python3-ruamel.yaml flac sox"
+checkdepends="$depends python3-Twisted"
```

Comment:
Please try adding `python3-pytest` to `checkdepends` as well, so the checks don't rely on deprecated setuptools functionality.